packages/devtools_app/lib/src/shared/table/table.dart

```diff
@@ -1521,7 +1521,9 @@ class _TableRowState<T> extends State<TableRow<T>>
           switch (displayTypeForIndex) {
             case _TableRowPartDisplayType.column:
               final current = columnIndexTracker;
-              columnIndexTracker++;
+              if (columnIndexTracker + 1 < widget.columns.length) {
+                columnIndexTracker++;
+              }
```

Comment:
why would we ever have more items in `rowDisplayParts` with type `_TableRowPartDisplayType.column` than we have columns? that seems like a bug with how `rowDisplayParts` is being generated

Reply:
it seems to be side effect of scroll overflow

Reply:
I dug into this a bit. Fix for this bug is here: https://github.com/flutter/devtools/pull/4606